cygnus-ngsi/conf/agent_ngsi.conf.template

```diff
@@ -257,6 +257,8 @@ cygnus-ngsi.sinks.mongo-sink.channel = mongo-channel
 #cygnus-ngsi.sinks.mongo-sink.data_model = dm-by-entity
 # how the attributes are stored, either per row either per column (row, column)
 #cygnus-ngsi.sinks.mongo-sink.attr_persistence = row
+# true if the metadata will be stored, false by default
```

Comment:
Probably this change adding a new parameter should be mirrored in .md documentation files that cite configuration (is there any of such fragments?) and also explained in accompanying text.

@pratid what do you think?